syz-fuzzer/workqueue.go

```diff
@@ -97,18 +97,14 @@ func (wq *WorkQueue) dequeue() (item interface{}) {
 	wq.mu.Lock()
 	wantCandidates := false
 	if len(wq.triageCandidate) != 0 {
-		last := len(wq.triageCandidate) - 1
-		item = wq.triageCandidate[last]
-		wq.triageCandidate = wq.triageCandidate[:last]
+		wq.triageCandidate, item = popWorkTriage(wq.triageCandidate)
```

Comment:
Since we have a separate function, it could check size and update the object as well:
```
if item := wq.triageCandidate.Pop(); item != nil {
  return item
}
```